src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEnginePublisher.java

```diff
@@ -213,6 +227,7 @@ public BuildStepMonitor getRequiredMonitorService() {
   @Extension
   public static final class DescriptorImpl extends BuildStepDescriptor<Publisher> {
     private static ComputeClient computeClient;
```

Comment:
Should this field still be static?

Reply:
No. I have another set of changes which removes this field and uses the non-static client factory to generate the compute client where needed, but didn't know if those changes were in scope.

Reply:
Included in this PR because it's a relatively small change.